randovania/interface_common/update_checker.py

```diff
@@ -84,6 +87,9 @@ def versions_to_display_for_releases(
             log = f"## {version.tag_name}\n\n{version.change_log}"
             all_change_logs[version.tag_name] = log
 
+            published_time = datetime.strptime(version.published_at, "%Y-%m-%dT%H:%M:%SZ")
```

Comment:
As much as I'd love to disrespect the terrible US date format, use the locale-dependant `%c` here.
It will make the tests a fun-experience though.

Reply:
I added the `%c` format in the latest commit, it looks like this:

![image](https://github.com/randovania/randovania/assets/118236986/626b09d1-242b-4140-9b10-9028f875dded)

I'm unfamiliar with this specific format, so I'm not sure if there's a way to further simplify this without splicing. Unless this is the result you wanted.

Reply:
Oh good catch. Forgot that one includes the time, which is very not relevant here.
Docs says %x is just the date.

Reply:
Okay, I updated the format

![image](https://github.com/randovania/randovania/assets/118236986/b040e003-8167-41df-b067-7a1961d6f925)